sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/CognitiveServicesManagementClientExtensions.cs

```diff
@@ -0,0 +1,119 @@
+    {
+            /// <summary>
+            /// Check available SKUs.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='location'>
+            /// Resource location.
+            /// </param>
+            /// <param name='skus'>
+            /// The SKU of the resource.
+            /// </param>
+            /// <param name='kind'>
+            /// The Kind of the resource.
+            /// </param>
+            /// <param name='type'>
+            /// The Type of the resource.
+            /// </param>
+            public static CheckSkuAvailabilityResultList CheckSkuAvailability(this ICognitiveServicesManagementClient operations, string location, IList<string> skus, string kind, string type)
+            {
+                return operations.CheckSkuAvailabilityAsync(location, skus, kind, type).GetAwaiter().GetResult();
+            }
+
+            /// <summary>
+            /// Check available SKUs.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='location'>
+            /// Resource location.
+            /// </param>
+            /// <param name='skus'>
+            /// The SKU of the resource.
+            /// </param>
+            /// <param name='kind'>
+            /// The Kind of the resource.
+            /// </param>
+            /// <param name='type'>
+            /// The Type of the resource.
+            /// </param>
+            /// <param name='cancellationToken'>
+            /// The cancellation token.
+            /// </param>
+            public static async Task<CheckSkuAvailabilityResultList> CheckSkuAvailabilityAsync(this ICognitiveServicesManagementClient operations, string location, IList<string> skus, string kind, string type, CancellationToken cancellationToken = default(CancellationToken))
+            {
+                using (var _result = await operations.CheckSkuAvailabilityWithHttpMessagesAsync(location, skus, kind, type, null, cancellationToken).ConfigureAwait(false))
+                {
+                    return _result.Body;
+                }
+            }
+
+            /// <summary>
+            /// Check whether a domain is available.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='subdomainName'>
+            /// The subdomain name to use.
+            /// </param>
+            /// <param name='type'>
+            /// The Type of the resource.
+            /// </param>
+            public static CheckDomainAvailabilityResult CheckDomainAvailability(this ICognitiveServicesManagementClient operations, string subdomainName, string type)
```

Comment:
Please write tests to cover this new operation thanks

Reply:
added